app/assets/stylesheets/stream_element.scss

```diff
@@ -28,12 +28,29 @@
     &:hover > .bd > .control-icons { background: $white; }
   }
   .thumbnail {
-    padding: 10px;
-    margin: 0 0 15px;
     background: $white;
     border-radius: 0;
     box-shadow: $card-shadow;
-    img.big_photo { max-height: 200px; }
+    height: 240px;
+    margin: 0 0 15px;
+    padding: 10px;
+
+    // Vertically align the image
+    text-align: center;
+    white-space: nowrap;
+
+    &::before {
+      content: '';
+      display: inline-block;
+      height: 100%;
+      vertical-align: middle;
+    }
```

Comment:
This is a famous technic (see http://stackoverflow.com/a/7310398) but it produces a small offset to the right here. Not sure why, maybe @svbergerem you have an idea?